core/arch/arm/plat-k3/drivers/ti_sci.c

```diff
@@ -81,6 +81,25 @@ static int ti_sci_setup_xfer(uint16_t msg_type, uint32_t msg_flags,
 	return 0;
 }
 
+/**
+ * ti_sci_send_request() - Send request to mailbox channel
+ *
+ * @xfer:	Transfer to initiate and wait for response
+ *
+ * Return: 0 if all goes well, else appropriate error message
+ */
+static inline int ti_sci_send_request(struct ti_sci_xfer *xfer)
```

Comment:
Could drop the `inline` attribute. Not strictly needed. But since there are such `inline` directive for many of the functions in this source file, maybe my comment is out of the scope of this P-R.

Reply:
Yeah, I think the code mostly is kept similar to how it is in tf-a and for driver fixes and implementations to come easily from optee -> tfa and vice versa. These things are kept simillar.